### ISSUE TITLE
Found a solution?

### ISSUE THREAD

**Issue by huzailhassan** [Alazoral/docx-import-error-repro#1]
Hello, were you able to find a solution to this problem? 

Thanks!

**Comment by Alazoral** [comment]
> Hello, were you able to find a solution to this problem? 
> 
> 
> 
> Thanks!

Yep! They issued a fix for it a while ago: https://github.com/dolanmiu/docx/issues/2255#issuecomment-1644651211
